jwst/stpipe/step.py

```diff
@@ -485,6 +484,25 @@ def finalize_result(self, result, reference_files_used):
         """
         pass
 
+    def remove_suffix(name):
+        """
+        Remove a known Step filename suffix from a filename
+        (if present).
+
+        Parameters
+        ----------
+        name : str
+            Filename.
+
+        Returns
+        -------
+        str
+            Filename with any known suffix removed.
+        str
+            Separator that delimited the original suffix.
+        """
+        return name
+
```

Comment:
This looks like a dummy implementation that does nothing.  Would it be better to return `NotImplemented`?

Reply:
This seemed like a reasonable default behavior since the original method returns the unmodified name if it doesn't contain any of the known substrings.  I notice now that I didn't return the second string here (the separator) so I should at least fix that.